src/plugins/wizard/opensearch_dashboards.json

```diff
@@ -0,0 +1,22 @@
+{
+  "id": "wizard",
+  "version": "1.0.0",
+  "opensearchDashboardsVersion": "opensearchDashboards",
+  "server": true,
+  "ui": true,
+  "requiredPlugins": [
+    "navigation",
+    "charts",
+    "data",
+    "opensearchDashboardsReact",
+    "opensearchDashboardsUtils",
+    "savedObjects",
+    "embeddable",
+    "expressions",
+    "dashboard",
+    "visualizations",
+    "opensearchUiShared",
```

Comment:
Yeah this may be unnecessary. Think we used it for the old index pattern selector. I'm hesitant to remove it at the last minute since I almost missed the other plugins too.